share/smack/lib/smack.c

```diff
@@ -1006,6 +1013,24 @@ void __SMACK_decls() {
 #undef D
 }
 
+// The size parameter represents number of bits that are being accessed.
```

Comment:
I am afraid I couldn't find any other function in LLVM which would give me the size of a datatype other than the function getTypeSizeInBits in class DataLayout. This function provides only bits. Moreover, the division happens in C not in boogie.